src/nodes.coffee

```diff
@@ -562,8 +628,8 @@ exports.Value = class Value extends Base
     lastProp instanceof Slice
 
   looksStatic: (className) ->
-    @base.value is className and @properties.length is 1 and
-      @properties[0].name?.value isnt 'prototype'
+    (@this or @base instanceof ThisLiteral or @base.value is className) and
+      @properties.length is 1 and @properties[0].name?.value isnt 'prototype'
```

Comment:
Can you explain this? I’m assuming we will add support for the `static` keyword, to explicitly define methods as static. How will that interact with this?